typescript/infra/src/core/multisend.ts

```diff
@@ -0,0 +1,74 @@
+import { ChainConnection, ChainName } from '@abacus-network/sdk';
+import { types } from '@abacus-network/utils';
+
+import { getSafe, getSafeService } from '../utils/safe';
+
+export abstract class MultiSend {
```

Comment:
im not sure about the usefulness of this versus just having extensions of ChainConnection and mapping calls through chainConnection.send but nbd

Reply:
Yeah I don't love this either.

The rationale was that for gnosis safe, you don't actually want to map calls through `chainConnection.send` b/c you want to batch them.

There's probably a better way than this though